### DISCUSSION TITLE
Powkiddy Q80 buttons not bound properly when selected as a PocketGo 2 V2 device in the kernel

### DISCUSSION THREAD

**Discussion by SeedyZ** [eduardofilo/RG350_adam_image#98] [Ideas & Requests]
I'm using the latest Adam Image on my Powkiddy Q80, and in the kernel file I set the device as the PocketGo V2. When I tested the inputs, I saw that the shoulder buttons weren't bound correctly. L1 and R2 weren't bound, and L2 was being seen as L1. I confirmed this through the input tester app. They don't register at all in any place in the system. I had previously had the device set as an RG350/P, and the buttons were bound correctly except for the PocketGo's Menu button (since the 350 doesn't have an equivalent).

Is there a way for me to bind the buttons myself, or is it an operating system level issue I can't easily fix?

**Comment by ridsama** [comment]
I have the same problem. Selecting PocketGo2 V2 messes up the Q80 buttons while RG350 works fine it seems. Not sure how much I will need the Menu button. The difference between the two is the system bin file. It's in binary so would have to compare them with an Hex editor or some sort.

**Comment by eduardofilo** [comment]
I'm sorry but I don't have the machine and it is something implemented at the system level, so those details should be referred to its developers.
